kinisi/parser.py

```diff
@@ -427,7 +428,7 @@ def _get_molecules(structure: Union["ase.atoms.Atoms", "pymatgen.core.structure.
 
 
 def _calculate_centers_of_mass(coords: VariableLikeType, weights: VariableLikeType,
-                               indices: VariableLikeType) -> VariableLikeType:
+                               indices: VariableLikeType, distance_unit:VariableLikeType) -> VariableLikeType:
```

Comment:
Can we double check that this is the correct type of for the distance unit?

Reply:
Sorry should be sc.Unit right